src/Stratis.Bitcoin.Features.RPC/RPCRouteHandler.cs

```diff
@@ -40,21 +40,21 @@ public async Task RouteAsync(RouteContext context)
         {
             Guard.NotNull(context, nameof(context));
 
-            var ms = new MemoryStream();
-            await context.HttpContext.Request.Body.CopyToAsync(ms);
-            context.HttpContext.Request.Body = ms;
-            ms.Position = 0;
-            JObject req = JObject.Load(new JsonTextReader(new StreamReader(ms)));
-            ms.Position = 0;
-            string method = (string)req["method"];
```

Comment:
This code I wrote made sure that the `Body` was still readable after `RouteAsync`, by default body are not readable again. (Except if calling `HttpRequest.EnabledRewind` and rewinding it, but I don't think it existed then)

Those if things don't break even without it, I guess this is good.

Reply:
I've just confirmed that there is no issue when calling the RPC methods through swagger.

Reply:
Thanks for the explanation, I though there might have been a valid reason for this construction but looks like we can simplify the code now.